x-pack/README.md

```diff
@@ -84,8 +84,9 @@ Jest integration tests can be used to test behavior with Elasticsearch and the K
 yarn test:jest_integration
 ```
 
-An example test exists at [test_utils/jest/integration_tests/example_integration.test.ts](test_utils/jest/integration_tests/example_integration.test.ts)
-
 #### Running Reporting functional tests
 
-See [here](test/reporting/README.md) for more information on running reporting tests.
+See [here](./test/functional/apps/dashboard/reporting/README.md) for more information on running reporting tests.
+
+#### Running Security Solution Cypress E2E/integration tests
+See [here](./plugins/security_solution/cypress/README.md) for information on running this test suite.
```

Comment:
@tylersmalley I added this link here since this file was [linked from the current docs](https://www.elastic.co/guide/en/kibana/current/development-tests.html), but it looks like that reference is gone in 7.11.

Any preference on where/how to reference this in the [7.11 documentation](https://www.elastic.co/guide/en/kibana/7.11/development-tests.html)?

Reply:
I think it's fine only backporting to 7.x/7.12.